dashboard/config/locales/en.yml

```diff
@@ -23,7 +23,9 @@ en:
   errors:
     messages:
       not_saved: 'Please correct the following errors:'
-      too_young: 'Your teacher must accept the updated Terms of Service before you can access this content. Your teacher can do this by logging into Code Studio and clicking the "Review Updated Terms" button near the top of Code Studio homepage.'
+      too_young: 'This content has age restrictions in place and is not available for younger students.'
+      teacher_must_accept_terms: 'Your teacher must accept the updated Terms of Service before you can access this content. Your teacher can do this by logging into Code Studio and clicking the "Review updated terms" button near the top of the Code Studio homepage.'
```

Comment:
Probably @poorvasingal 's thinking was that the student would just show their screen to the teacher? But I kind of agree with you. Would suggest something like "This content has age restrictions in place and is not available for younger students. Ask your teacher to accept the latest Terms of Service to give you access to this content."

What if a teacher wants to say no, is that awkward?